test/specs/buffer-with-count.coffee

```diff
@@ -0,0 +1,107 @@
+{stream, prop, send, Kefir} = require('../test-helpers.coffee')
+
+describe 'bufferWithCount', ->
+
+  describe 'stream', ->
+
+    it 'should return stream', ->
+      expect(stream().bufferWithCount(1)).toBeStream()
+
+    it 'should activate/deactivate source', ->
+      a = stream()
+      expect(a.bufferWithCount(1)).toActivate(a)
+
+    it 'should be ended if source was ended', ->
+      expect(send(stream(), ['<end>']).bufferWithCount(1)).toEmit 
+      []
+      ['<end:current>']
```

Comment:
> no language should have let me do that :)

btw, I'm thinking of moving away from CoffeeScript, but it's just a lot of code to migrate...